public/rulesets/default/2_3_2_Content_Type

```diff
@@ -0,0 +1,24 @@
+rules:  
+  - id: "2.3.2"
+    title: "Content-Type: Default Media Type"
+    message: "If the request or response body is present, the default media type must be application/json with charset=UTF-8 or application/bson."
+    option: Mandatory
+    location: path
+    element:
+  	- requestBody
+  	- responses
+    call:
+  	function: mediaTypeCheck
+  	functionParams:
+  	  headers:
+  		- "application/json; charset=UTF-8"
```

Comment:
please remove charset as this is not present in opeapi specification
and add headers is key value format:
```
- Content-Type:
    "application/json"
```
https://swagger.io/docs/specification/v3_0/describing-parameters/#header-parameters

We probably support only key in this version as OpenAPI doesn't support values for now (https://github.com/OAI/OpenAPI-Specification/discussions/3827) but In future we might support different syntax extended by content which will introduce also values.
      content:
        'application/json'